### DISCUSSION TITLE
The word "Blocked" being misunderstood

### DISCUSSION THREAD

**Discussion by sooluh** [angeloanan/kominfo-d-#15] [General]
I see in some media (especially TikTok), many people may misunderstand. Isn't the word :sparkles:MENKOMINFO:sparkles: not blocked? but restricted? Looks like we need to change the word "Blocked" to "Restricted" to avoid misunderstanding.

This should have been made in the Discussion feature, but the discussion feature is not enabled here, so I put it in Issues.

Let's talk about this!
Thank you :)

**Comment by angeloanan** [comment]
Heya, I understand where you're coming from.

The wording issue has been brought up several times in the past to me in Twitter DMs and in [another GitHub issue](https://github.com/angeloanan/kominfo-d-/issues/7).

Though, I'm insisting on using the word `Blocked` as the gov'd wrote in [Permenkominfo 5/2020](https://peraturan.bpk.go.id/Home/Details/203049/permenkominfo-no-5-tahun-2020) Bab II Bagian ketiga Pasal 7 Nomor 2:

![Dalam hal PSE Lingkup Privat tidak melakukan pendaftaran sebagaimana dimaksud pada ayat (1) huruf a, Menteri memberikan sanksi administratif berupa Pemutusan Akses terhadap Sistem Elektronik (access blocking).](https://user-images.githubusercontent.com/2129163/180936860-e62df9c2-653e-40f0-a741-e3d50f8d19bb.png)

To be honest, I am very open to changing this. I just need a compelling reason which will make me want to change the wording.

Do not hesitate to follow up!

**Reply by sooluh** [reply]
Oh my bad, apparently there have been previous issues here. It's true that Permenkominfo 5/2020 says that, but some people still think it's restricted (it seems at that time I found a tweet saying that MENKOMINFO said "Restricted" not "Blocked").

Fine then, I'll follow it. By the way, thank you, so far it's still fine. Haha :sweat_smile:
